src/components/Sum/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export const Sum = () => ();
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```
Here it also can be in one line if the amount of props is less than 3